From 8ae853472a61d099b32a5068364cf4d75fcaac04 Mon Sep 17 00:00:00 2001
From: Marc Mutz <marc.mutz@qt.io>
Date: Thu, 2 Oct 2025 15:10:14 +0200
Subject: [PATCH 382/553] QWizardHeader: mark the ctors explicit

Found while working in the vicinity.

Amends the start of the public history.

Pick-to: 6.8 6.5
Change-Id: I2df202fd18a0b80412b5f689fd0b454106adb4d8
Reviewed-by: Richard Moe Gustavsen <richard.gustavsen@qt.io>
(cherry picked from commit e85f6d6c5c217f8d316efa67c8de6effbbbcccb3)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 src/widgets/dialogs/qwizard.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/widgets/dialogs/qwizard.cpp b/src/widgets/dialogs/qwizard.cpp
index 7473fcd2963..17859b34069 100644
--- a/src/widgets/dialogs/qwizard.cpp
+++ b/src/widgets/dialogs/qwizard.cpp
@@ -249,9 +249,11 @@ class QWizardHeader : public QWidget
 protected:
     enum RulerType { Ruler };
 
+    explicit
     inline QWizardHeader(RulerType /* ruler */, QWidget *parent = nullptr)
         : QWidget(parent) { setFixedHeight(2); }
 public:
+    explicit
     QWizardHeader(QWidget *parent = nullptr);
 
     void setup(const QWizardLayoutInfo &info, const QString &title,
-- 
2.51.2

